common/membership/ringpop/factory.go

```diff
@@ -70,6 +70,8 @@ type factory struct {
 	monOnce sync.Once
 }
 
+var errMalformedBroadcastAddress = fmt.Errorf("ringpop config malformed `broadcastAddress` param")
```

Comment:
nit: you don't need formatted string

```suggestion
var errMalformedBroadcastAddress = errors.New("ringpop config malformed `broadcastAddress` param")
```